### ISSUE TITLE
Convert InplaceABN to standard Batch Normalization

### ISSUE THREAD

**Issue by keivanmoazami** [mapillary/inplace_abn#208]
I want to serve model with InPlaceABN layer in triton server but pytorch jit and export to onnx not work with inplace-abn. 
Can I convert inplace-abn to standard batch normalization layer? if yes, how can I do that ?
thanks

**Comment by songzijiang** [comment]
I have the same requirement, if you find a way to convert InplaceABN, please remind me. thank you

**Comment by keivanmoazami** [comment]
This question has already been answered
https://github.com/mapillary/inplace_abn/issues/163#issuecomment-802449498

**Comment by wolterlw** [comment]
@keivanmoazami, I have tried to follow the conv-bn fusion, but the output of original and "fused" models to not match. Did you do anything else to the code provided?
I'm using `InplaceABNSync` if that's important.
